content/en/docs/reference/config/metrics/index.md

```diff
@@ -113,3 +111,9 @@ For TCP traffic, Istio generates the following metrics:
     destination_canonical_service
     destination_canonical_revision
     {{< /text >}}
+
+*   **Destination Cluster**: Name of the cluster for the destination workload.
```

Comment:
Should we match the wording of all the other items?

"This identifies the cluster of the destination workload."

Same for source.

Reply:
Sure. This was just copying over the dropped text from before, but we should update it to match.